linkerd/proxy/server-policy/src/lib.rs

```diff
@@ -144,18 +146,25 @@ pub mod proto {
                     timeout: _,
                     http_routes: _,
                     http_local_rate_limit,
-                }) => http_local_rate_limit.unwrap_or_default().into(),
+                }) => http_local_rate_limit.unwrap_or_default(),
                 api::proxy_protocol::Kind::Http1(api::proxy_protocol::Http1 {
                     routes: _,
                     local_rate_limit,
                 })
                 | api::proxy_protocol::Kind::Http2(api::proxy_protocol::Http2 {
                     routes: _,
                     local_rate_limit,
-                }) => local_rate_limit.unwrap_or_default().into(),
+                }) => local_rate_limit.unwrap_or_default(),
                 _ => Default::default(),
             };
 
+            let local_rate_limit_meta = local_rate_limit
+                .clone()
+                .metadata
+                .map(TryInto::try_into)
+                .transpose()?
+                .unwrap_or(Meta::Default { name: "".into() });
```

Comment:
this empty name gives me pause. if we already have a `Meta` object in `local_rate_limit`, why not just use that? `Option<T>` values can be encoded as labels, (_see below_), so maybe we can work with optional metadata without creating a default, unnamed `Meta` when none exists.

what do you think?

https://docs.rs/prometheus-client/latest/prometheus_client/encoding/trait.EncodeLabelValue.html#impl-EncodeLabelValue-for-Option%3CT%3E